stable/coredns/templates/deployment.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     {{- end }}
     app.kubernetes.io/name: {{ template "coredns.name" . }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
I think this is ok and generally you can merge it, but if there is a possibility that we are going to add this to other objects I would suggest that we create a template and use it everywhere.